From 88c9553cf4a3e563d49196f088f4f64465db822c Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Sat, 29 Oct 2016 20:56:46 +0200
Subject: [PATCH] udev: add uaccess tag

In more modern systems that use systemd-logind, plugdev group is no
longer used to arbitrate access to the devices. Instead, the uaccess tag
on the device instructs logind to make it available to whichever seat is
active.

Let's keep both rules for the time being.
---
 udev/rules.d/80-minipro.rules | 1 +
 1 file changed, 1 insertion(+)

diff --git a/udev/rules.d/80-minipro.rules b/udev/rules.d/80-minipro.rules
index b1976fc..1b32040 100644
--- a/udev/rules.d/80-minipro.rules
+++ b/udev/rules.d/80-minipro.rules
@@ -1 +1,2 @@
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="04d8", ATTRS{idProduct}=="e11c", TAG+="uaccess"
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="04d8", ATTRS{idProduct}=="e11c", MODE="0660", GROUP="plugdev"
